### PR TITLE
update branding for mono nuget transport packages

### PR DESCRIPTION
**Pull request by Anipik**

### CODE REVIEW

File: src/mono/Directory.Build.props [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
   <PropertyGroup>
     <!-- Central place to set the versions of all nuget packages produced in the repo -->
-    <PackageVersion Condition="'$(PackageVersion)' == ''">5.0.0</PackageVersion>
+    <PackageVersion Condition="'$(PackageVersion)' == ''">5.0.1</PackageVersion>
```
Comment (mmitche) [inline src/mono/Directory.Build.props:27]: Should this be here at all? It seems like it should just pull from the eng/Versions.props locations.

Comment (Anipik) [reply src/mono/Directory.Build.props]: removed

**Review by safern — APPROVED** [review]
What about in master? We should probably backport this change to master.

**Comment by Anipik** [comment]
> What about in master? We should probably backport this change to master.

yes will backport there as well

**Pull request merged**